suite-common/suite-types/src/messageSystem.ts

```diff
@@ -0,0 +1,158 @@
+/**
+ * DO NOT MODIFY BY HAND! This file was automatically generated.
+ * Instead, modify the original JSONSchema file in /message-system/schema, and run yarn build:libs.
+ */
+
+/**
+ * ISO 8601 date-time format.
+ */
+export type DateTime = string;
+export type Version = string | string[];
+export type Model = 'T' | '1' | 'T2B1';
+export type FirmwareRevision = string;
+export type FirmwareVariant = '*' | 'bitcoin-only' | 'regular';
+/**
+ * Eligible authorized vendors.
+ */
+export type Vendor = '*' | 'trezor.io';
```

Comment:
Note: If I remember correctly, I wanted to extend this by a vendor trezor used in the past. So if you create some followup issue with some notes, you can add this there.

Reply:
We can easily add it if we need to target it - with the current code it would be backward compatible.